tests/console/wavpack.pm

```diff
@@ -35,8 +35,12 @@ sub run {
     # development module needed for dependencies, released products are tested with sdk module
     if (is_sle) {
         if (get_var('BETA')) {
-            my $sdk_repo = is_sle('15+') ? get_var('REPO_SLE_MODULE_DEVELOPMENT_TOOLS') : get_var('REPO_SLE_SDK');
-            zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => "SDK";
+            if (is_sle('15-sp1+')) {
+                add_suseconnect_product('sle-module-desktop-applications');
+            } else {
+                my $sdk_repo = is_sle('15+') ? get_var('REPO_SLE_MODULE_DEVELOPMENT_TOOLS') : get_var('REPO_SLE_SDK');
+                zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => "SDK";
```

Comment:
I'm not entirely sure but:
```suggestion
                add_suseconnect_product(get_addon_fullname('sdk'));
```
should work, and you can remove the other line too..